src/test/librbd/CMakeLists.txt

```diff
@@ -133,6 +133,13 @@ if(WITH_RBD_RWL)
      cache/pwl/test_WriteLogMap.cc)
 endif(WITH_RBD_RWL)
 
+if(WITH_RBD_SSD_CACHE)
+   set(unittest_librbd_srcs
+     ${unittest_librbd_srcs}
+     cache/pwl/test_mock_SSDWriteLog.cc
+     cache/pwl/test_WriteLogMap.cc)
```

Comment:
Is this being included twice if `RWL` mode is enabled?

Reply:
Done